newparp/templates/email/reset.html

```diff
@@ -0,0 +1,10 @@
+{% extends "email/base.html" %}
+{% block content: %}
+  <h2 style="margin: 50px 50px 20px; color: #eee; font-size: 2em; text-align: center; line-height: normal; text-transform: uppercase;">Verify your e-mail address</h2>
+  <div style="box-sizing: border-box; max-width: 650px; margin: 20px auto; padding: 20px 25px; background-color: #eee;">
+    <p style="margin-top: 0;">If you've forgotten your password, please visit the following URL to reset it: {{url_for("reset_password", _external=True, _scheme="https", user_id=user.id, email_address=email_address, token=email_token)|safe}}.</p>
```

Comment:
Ideally this should be a link so people can just click it rather than cut and pasting. Also people might mistake the full stop for part of the URL.

Reply:
Alright, I will look into changing that to a link I will make these changes alongside the others I need to implement.